NumerGuess.py

```diff
@@ -0,0 +1,16 @@
+repeat = "true"
+while repeat == "true":
+	guess = raw_input("guess a number (1-10) ")
+	if guess == 3:
```

Comment:
The ```3``` is a [magic number](http://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants).  Magic numbers make me sad.